### ISSUE TITLE
control network `to_isis` and `from_isis` expect different column names in the df

### ISSUE THREAD

**Issue by jessemapel** [USGS-Astrogeology/plio#121]
The `from_isis` method creates a df with column names that match the ISIS protobuf spec. The `to_isis` method expects a df with different column names. This means you cannot do the following:

```
df = from_isis(`some.net`)
do things to df
to_isis(df, `some_modified.net`)
```

**Comment by jessemapel** [comment]
This may be the cause of #111

**Comment by jlaura** [comment]
Labelled as a bug since we should be able to roundtrip inside of plio. Consumers of plio that need to do formatting should be doing that formatting locally.

**Comment by acpaquette** [comment]
@jessemapel Looks like the case you outlined in the original issue was fixed by #131. However the split between measure columns and point columns outlined in #122 may no longer be present

**Comment by jessemapel** [comment]
That is a problem then. There are fields on the measures and points that have the same name. if they don't have the prefixes any more then things are not going to write out correctly.
